src/controllers/dashboardpage.js

```diff
@@ -398,13 +395,12 @@ define(["datetime", "events", "itemHelper", "serverNotifications", "dom", "globa
             }
             if (showTranscodingInfo) {
                 var line = [];
-
                 if (session.TranscodingInfo) {
                     if (session.TranscodingInfo.Bitrate) {
                         if (session.TranscodingInfo.Bitrate > 1e6) {
                             line.push((session.TranscodingInfo.Bitrate / 1e6).toFixed(1) + " Mbps");
                         } else {
-                            line.push(Math.floor(session.TranscodingInfo.Bitrate / 1e3) + " kbps");
+                            line.push(Math.floor(session.TranscodingInfo.Bitrate / 1e3) + " Kbps");
```

Comment:
I understand that per ISO 1 Kb is 1000 bytes, but maybe move to 1 KiB = 1024 bytes here somehow?..

Reply:
Generally Kbps is used for any kind of streaming measurement which is why I kept the label the same as before.